scripts/cli/src/commands/helpers/identity_helper.ts

```diff
@@ -0,0 +1,34 @@
+import {nonces, sendTransaction} from "../util/init"
+import { topUpIdentityBalance } from "./balance_helper"
+import { IKeyringPair } from '@polkadot/types/types'
+import { ApiPromise } from "@polkadot/api"
+import { SigningItem } from "../types"
+
+export default async function createIdentities(api: ApiPromise, accounts: IKeyringPair[], alice: IKeyringPair) {
+    return await createIdentitiesWithExpiry(api, accounts, alice, []);
+};
+
+const createIdentitiesWithExpiry = async function(api: ApiPromise, accounts: IKeyringPair[], alice: IKeyringPair, expiries: SigningItem[]) {
```

Comment:
We should use `cddProvider` instead of `alice` for the variable name.

Reply:
I will make the change.